src/cargo/util/toml/targets.rs

```diff
@@ -1023,9 +1040,12 @@ Cargo doesn't know which to use because multiple target files found at `{}` and
 
 /// Returns the path to the build script if one exists for this crate.
 #[tracing::instrument(skip_all)]
-pub fn normalize_build(build: Option<&StringOrBool>, package_root: &Path) -> Option<StringOrBool> {
+pub fn normalize_build(
+    build: Option<&StringOrBool>,
+    package_root: &Path,
+) -> CargoResult<Option<StringOrBool>> {
```

Comment:
Does this still need to return `CargoResult` if we switched the error to an `expect`?

Reply:
Oh, yes!